src/testutils/configurable_blockchain_tests.rs

```diff
@@ -1,8 +1,9 @@
+use assert_matches::assert_matches;
```

Comment:
Same unused warning here.. `assert_matches` not used..

Reply:
fixed